<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 0);
background-color: rgb(255, 255, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="entrada">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="borrar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limpiar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enviar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>enviar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>INGRESE SU DNI POR FAVOR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="msjturno">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TURNOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Caja">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Caja</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Licencia_conducir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Licencia de conducir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Catastro">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Catastro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>420</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Comercio">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Comercio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Agua">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Agua</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Propiedades">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Propiedades</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Automotor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Tasas del automotor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Preferencial">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string>Servicio Preferencial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Otros">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
border-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);
border: 2px solid #8f8f91;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string>Otros</string>
    </property>
   </widget>
   <widget class="QLabel" name="ticket">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fecha">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hora">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="inicio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>inicio</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
